lib/csv.rb

```diff
@@ -2260,25 +2260,56 @@ def header_convert(name = nil, &converter)
 
   include Enumerable
 
+  # :call-seq:
+  #   csv.each -> self
```

Comment:
Could you change the return value to "undefined"?

```suggestion
  #   csv.each -> enumerator
  #   csv.each {|row| ...}
```